src/main/java/com/google/api/codegen/discovery/transformer/nodejs/NodeJSSampleTypeNameConverter.java

```diff
@@ -92,18 +91,12 @@ public TypeName getTypeNameForElementType(TypeInfo typeInfo) {
   @Override
   public TypedValue getZeroValue(TypeInfo typeInfo) {
     // Don't call getTypeName; we don't need to import these.
-    if (typeInfo.isMap()) {
-      return TypedValue.create(new TypeName("Object"), "{}");
-    }
     if (typeInfo.isArray()) {
       return TypedValue.create(new TypeName("Array"), "[]");
     }
     if (PRIMITIVE_ZERO_VALUE.containsKey(typeInfo.kind())) {
       return TypedValue.create(getTypeName(typeInfo), PRIMITIVE_ZERO_VALUE.get(typeInfo.kind()));
     }
-    if (typeInfo.isMessage()) {
```

Comment:
So all these things that I removed shouldn't happen, I removed them straight up because I want an exception raised if we encounter these conditions:

If I'm not mistaken, fields can be objects and arrays but they cannot be maps. Also, if a method has a request body, the field cannot be a message. I opted to fail entirely if we do happen to see that since these code paths don't get tested at all, and probably cannot be tested.